lib/vif-plug-providers/representor/vif-plug-representor.c

```diff
@@ -381,7 +390,11 @@ port_table_delete_entry(struct port_table *tbl,
     }
 }
 
+#ifdef HAVE_UDEV
```

Comment:
This ifdef should probably be moved down a line as presence of libudev does not dictate devlink usage.